crates/sui-json-rpc/src/read_api.rs

```diff
@@ -121,6 +125,64 @@ impl SuiRpcModule for ReadApi {
 
 #[async_trait]
 impl RpcFullNodeReadApiServer for FullNodeApi {
+    async fn get_move_function_arg_types(
```

Comment:
Can you also add sample data generation for this endpoint: https://github.com/MystenLabs/sui/blob/bfb41f2d36d4cd4d080c91253f66583bb212655e/crates/sui-open-rpc/src/generate_json_rpc_spec.rs#L148 ?